CHANGELOG/CHANGELOG-3.4.md

```diff
@@ -13,6 +13,7 @@ Previous change logs can be found at [CHANGELOG-3.3](https://github.com/etcd-io/
 ### Dependencies
 
 - [Scripts/build-binary.sh: use `buildvcs=false` to avoid having a pseudo-version reported by `go version`](https://github.com/etcd-io/etcd/pull/20950)
```

Comment:
pls add it under Dependencies, refer to https://github.com/etcd-io/etcd/blob/main/CHANGELOG/CHANGELOG-3.5.md#dependencies

Reply:
I should raise this comment to other two files.